screenpy/actions/either.py

```diff
@@ -86,6 +75,23 @@ def describe(self) -> str:
 
         return f"Either {try_summary} or {except_summary}"
 
+    def perform_as(self, the_actor: Actor) -> None:
```

Comment:
To avoid missing anything, I would separate moving sections of code from the PR which contains changes.

Reply:
Yeah, you're right. I just didn't think i'd remember if i put it off. I'll move it back.